spec/support/blueprints.rb

```diff
@@ -208,6 +208,33 @@ def first_or_make_site_unless_manufacturer(institution)
   samples { [Sample.make(:filled, box: object, institution: object.institution, site: object.site) ] }
 end
 
+Box.blueprint(:LOD) do
+  institution { object.site.try(&:institution) || Institution.make }
+  purpose { "LOD" }
+  samples { [
+    Sample.make(institution: object.institution, date_produced: Date.today),
+    Sample.make(institution: object.institution, date_produced: Date.today),
+  ] }
+end
+
+Box.blueprint(:Variants) do
+  institution { object.site.try(&:institution) || Institution.make }
+  purpose { "Variants" }
+  samples { [
+    Sample.make(institution: object.institution, date_produced: Date.today),
+    Sample.make(institution: object.institution, date_produced: Date.today),
+  ] }
+end
+
+Box.blueprint(:Challenge) do
+  institution { object.site.try(&:institution) || Institution.make }
+  purpose { "Challenge" }
+  samples { [
+    Sample.make(institution: object.institution, date_produced: Date.today),
+    Sample.make(institution: object.institution, date_produced: Date.today),
+  ] }
+end
+
```

Comment:
I'm wondering if it makes sense to define purpose specific blueprints when they don't really implement the purpose box? The samples are just stubbed.
You would get pretty much the same result using the generic blueprint with a purpose attribute (`Box.make(purpose: "Challenge)`). Only diff would be the number of samples, but I think we can simply increase that to two in the default blueprint.